packages/radio-group/test/visual/lumo/radio-group.test.js

```diff
@@ -92,19 +92,19 @@ describe('radio-group', () => {
       document.documentElement.removeAttribute('dir');
     });
 
-    it('RTL basic', async () => {
+    it('basic', async () => {
```

Comment:
Let's revert test name changes in this file, there is no need for them.

Reply:
Good point! Reverted.